### DISCUSSION TITLE
--fmt, --check inconsistency; rename to --format ?

### DISCUSSION THREAD

**Discussion by simonmichael** [casey/just#1801] [Ideas]
Somewhat related to #1765. --fmt is abbreviated, --check is not. This seems like a trivial thing but I find it adds an ongoing small cognitive load with every justfile I work with. I wrap these in recipes for convenience, so there are also corresponding recipe names. Sometimes I hide these mundane recipes with an underscore, now the name choices double again.  In each new project or folder with a justfile I'm wondering is it chk ? check ? fmt ? format ? Have I kept every justfile consistent ?

So, how renaming to "--format", perhaps keeping --fmt as a hidden alias ? I feel the full spelling would be best overall, for clarity and consistency with the other flags.

**Comment by casey** [comment]
I personally like the abbreviated flags, but I think have the un-abbreviated flags as aliases is reasonable. Added in #1802.

**Reply by simonmichael** [reply]
Thanks!

When you say "flags" - isn't `--fmt` just's only abbreviated flag ?

**Reply by casey** [reply]
`--init` is also abbreviated, so I added `--initialize` as an alias.
